tests/lib/nm/ipv6_test.py

```diff
@@ -54,10 +57,15 @@ def test_create_setting_with_ipv6_disabled(NM_mock):
         config={InterfaceIPv6.ENABLED: False}, base_con_profile=None
     )
 
-    assert (
-        ipv6_setting.props.method == NM_mock.SETTING_IP6_CONFIG_METHOD_IGNORE
+    assert ipv6_setting == NM_mock.SettingIP6Config.new.return_value
+    disable_method = (
+        NM_mock.SETTING_IP6_CONFIG_METHOD_DISABLED
+        if nm.nmclient.can_disable_ipv6()
+        else NM_mock.SETTING_IP6_CONFIG_METHOD_IGNORE
     )
 
+    assert ipv6_setting.props.method == disable_method
```

Comment:
The tests are failing here.
The problem is that the `_can_disable_ipv6` is global and resolved before the `NM_mock` fixture is doing its job.

I quick fix is to mock `nmclient.can_disable_ipv6` and set it to return `True`.